src/main/java/com/iexec/sms/secret/web2/Web2SecretService.java

```diff
@@ -38,7 +38,7 @@ protected Web2SecretService(EncryptionService encryptionService,
     }
 
     public Optional<Web2Secret> getSecret(String ownerAddress, String secretAddress) {
-        return web2SecretRepository.findByOwnerAddressAndAddressAllIgnoreCase(ownerAddress, secretAddress);
+        return web2SecretRepository.find(ownerAddress, secretAddress);
```

Comment:
Where did the lowercase go?

Reply:
Good catch! Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/140/commits/1157f62e9ab28cd0bb4fb02dea2b141ebda323a0.